patches/chrome-browser-resources-settings-settings_menu-settings_menu.html.patch

```diff
@@ -1,28 +1,232 @@
 diff --git a/chrome/browser/resources/settings/settings_menu/settings_menu.html b/chrome/browser/resources/settings/settings_menu/settings_menu.html
-index 2fcea63a9bf9bf0400640d3ec8b6277087814bab..892f1693a46f0f7df9172ecd7be2dc65fe263b7f 100644
+index 2fcea63a9bf9bf0400640d3ec8b6277087814bab..d116ae93881505cfc720fdeb6b334efb91fb17ec 100644
 --- a/chrome/browser/resources/settings/settings_menu/settings_menu.html
 +++ b/chrome/browser/resources/settings/settings_menu/settings_menu.html
-@@ -148,6 +148,10 @@
-         $i18n{defaultBrowser}
+@@ -10,44 +10,69 @@
+ <link rel="import" href="../route.html">
+ <link rel="import" href="../settings_shared_css.html">
+ 
++<if expr="not _google_chrome">
++<link rel="import" href="../brave_settings_menu/icons.html">
++</if>
++
+ <dom-module id="settings-menu">
+   <template>
+     <style include="settings-shared">
+       :host {
+-        display: block;
```

Comment:
we need to look at other ways to do this, this patch is going to be a nightmare to maintain